### ISSUE TITLE
User remains logged in on browser close

### ISSUE THREAD

**Issue by scripttease** [scripttease/ELSWC#6]
If tab or browser is closed with user logged in, they are remembered for the next time it is opened. Need to add automatic log out on tab or browser close

**Comment by lpil** [comment]
Title is a bit confusing. Is their remaining logged in the desired behaviour or not

**Comment by scripttease** [comment]
Nope. It is BAD. This is a bad thing. OK what would you call it. I'm all out of snappy titles today

**Comment by lpil** [comment]
Generally the title is the intended behaviour. Can use words like "should" to reduce ambiguity.
